app/views/settings/_page_body.html.erb

```diff
@@ -1,4 +1,4 @@
-<div id="settings-page-body-mobile" class="my-6 flex gap-8 rounded border-b p-8 border-line-colour box-shadow-medium">
+<div id="settings-page-body-desktop" class="flex gap-8 border-b p-8 border-line-colour ">
```

Comment:
change id to  "settings-page-body"

change lines in update.turbo_stream.erb from views/settings and make sure the code works on updating on both mobile and web

Reply:
id changed ,also corresponding changes done in  update.turbo_stream.erb from views/settings